db/migrate/20170805061539_create_mail_logs.rb

```diff
@@ -0,0 +1,15 @@
+class CreateMailLogs < ActiveRecord::Migration
+  def change
+    create_table :mail_logs do |t|
+      t.references :user, index: true
+      t.string :name
+      t.string :email
```

Comment:
I'm very sorry, just couple of tips regarding this model.

It looks like your creating a data duplication here (it seems from a code i see, that you log mails for existing users). I'm not sure if this is a best practice or a secure one. Can you just save user.id in here?